src/components/Text/Text.spec.tsx

```diff
@@ -30,10 +30,24 @@ describe('Text', () => {
         });
     });
 
+    describe('if the `secondary` property is set', () => {
+        it('has a dim color', () => {
+            expect(render(<Text secondary />).container.firstChild).toHaveStyle(`
+                color: ${Colors.AUTHENTIC_BLUE_550};
+            `);
+        });
+
+        it('has a dimmer color if inverted', () => {
+            expect(render(<Text secondary inverted />).container.firstChild).toHaveStyle(`
+                color: ${Colors.AUTHENTIC_BLUE_350};
+            `);
+        });
+    });
+
     describe('if the disabled property is set', () => {
         it('has the disabled color', () => {
             expect(render(<Text disabled />).container.firstChild).toHaveStyle(`
-                color: ${Colors.AUTHENTIC_BLUE_350};
+                color: ${Colors.AUTHENTIC_BLUE_200};
```

Comment:
For me, the test should check the final value assigned to avoid occasional changes in the semantic colors assignment. 

But I don't have a strong opinion here

Reply:
@nlopin the thing is that with semantic colors they should be the primary source of truth, rather then the brand colors (`Colors.AUTHENTIC_BLUE_350`). Because the brand colors might change, but the semantic should stay, because it reflects the purpose of the element (text, in this case). In FREE NOW there was, already, several color rebranding and, by the way, soon will be another round. So in best we should link all the things to semantic colors, and then, if we need to do that, we can change theirs values in one single place.

Reply:
Makes sense, I'll change the test to use the semantic colors